deisctl/client/client.go

```diff
@@ -240,6 +241,20 @@ Usage:
 	return cmd.ListUnits(c.Backend)
 }
 
+func (c *Client) Machines(argv []string) error {
+	usage := `Enumerate the current hosts in the cluster
```

Comment:
Makes sense.  We don't need to be just like them, but this is enough precedent to probably leave it be.  :smile: